testcases/kernel/syscalls/fchmod/fchmod01.c

```diff
@@ -163,7 +163,7 @@ void setup(void)
 	sprintf(fname, "tfile_%d", getpid());
 	if ((fd = open(fname, O_RDWR | O_CREAT, 0700)) == -1)
 		tst_brkm(TBROK | TERRNO, cleanup, "open failed");
-	else if (write(fd, &buf, strlen(buf)) == -1)
+	else if (write(fd, (const void *)buf, strlen(buf)) == -1)
```

Comment:
In 32-bits,if we use "&buf", it will can't run.So can it been merged now,or is there also some issues?